### PR TITLE
updating contributors

### PR DESCRIPTION
**Pull request by sadiqevani**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-  "version": "3.0.0",
+  "version": "3.1.0",
   "name": "@aeternity/aepp-components",
   "main": "dist/aepp.components.js",
   "description": "Reusable Vue Components for aeternity aepp projects",
   "homepage": "https://github.com/aeternity/aepp-components#readme",
-  "author": "Max Dörfler <max@apeunit.com> (https://github.com/muxe)",
+  "contributors": [
+    "Max Dörfler <max@apeunit.com> (https://github.com/muxe)",
+    "Denis Davidyuk <denis@apeunit.com> (https://github.com/davidyuk",
```
Comment (davidyuk) [inline package.json]: I don't have an email at apeunit.com, so:
```suggestion
    "Denis Davidyuk <denis_davidyuk@hotmail.com> (https://github.com/davidyuk)",
```

File: package.json [hunk 2]
```diff
@@ -1,10 +1,14 @@
 {
-  "version": "3.0.0",
+  "version": "3.1.0",
```
Comment (davidyuk) [inline package.json]: Actually, the third version is not released yet, let's postpone version changing

File: package.json (hunk 1 above)
Comment (sadiqevani) [reply package.json]: fixed

File: package.json (hunk 2 above)
Comment (sadiqevani) [reply package.json]: fixed

**Review by davidyuk — APPROVED** [review]

**Pull request merged**